website/docs/r/container_group.html.markdown

```diff
@@ -41,6 +41,7 @@ resource "azurerm_container_group" "aci-helloworld" {
   resource_group_name = "${azurerm_resource_group.aci-rg.name}"
   ip_address_type     = "public"
   os_type             = "linux"
+  restart_policy      = "Always"

```

Comment:
_minor_ since this is defaulted, I don't think we need to specify it in the example

Reply:
Sure.